README.md

```diff
@@ -23,9 +23,8 @@ You can find the deployed project at [CoderHeroes](https://a.coderheroes.dev/log
 | [Therman Mcmillan](https://github.com/mcmillanj) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/mcmillanj)        | [Mark Escosura](https://github.com/mark-escosura) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/mark-escosura) | [Priscila Monteiro](https://github.com/PriscilaMonteiro) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/PriscilaMonteiro) | [Rose Gabriely](https://github.com/RoseGabriely) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/RoseGabriely) |
-| [<img src="https://avatars.githubusercontent.com/u/83845399?v=4" width = "200" />](https://github.com/waylonturbes)                            | [<img src="https://avatars.githubusercontent.com/u/87030206?v=4" width = "200" />](https://github.com/krisf451)                              | [<img src="https://avatars.githubusercontent.com/u/56285755?v=4" width = "200" />](https://github.com/seanmp3)                                         |
-| [Waylon Turbes](https://github.com/waylonturbes) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/waylonturbes)     | [Kristian Fulkerson](https://github.com/krisf451) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/krisf451)      | [Sean Mahan](https://github.com/seanmp3) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/seanmp3)                          |
- |
+| [<img src="https://avatars.githubusercontent.com/u/83845399?v=4" width = "200" />](https://github.com/waylonturbes)                            | [<img src="https://avatars.githubusercontent.com/u/87030206?v=4" width = "200" />](https://github.com/krisf451)                              | [<img src="https://avatars.githubusercontent.com/u/56285755?v=4" width = "200" />](https://github.com/seanmp3)                                         | [<img src="https://avatars.githubusercontent.com/u/98560786?v=4" width="200"> ](https://github.com/theecolya) |
+| [Waylon Turbes](https://github.com/waylonturbes) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/waylonturbes)     | [Kristian Fulkerson](https://github.com/krisf451) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/krisf451)      | [Sean Mahan](https://github.com/seanmp3) [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/seanmp3)                          |[Colya Foxworth](https://github.com/theecolya) <img src="https://github.com/favicon.ico" width="15">
```

Comment:
I am not sure if this matters but do you need a bracket in front of the last <img tag?

Reply:
@MaxT6 I think that's what makes it a link in markdown

Reply:
I went down a bit too far down a rabbit hole but something [like this](https://lukasznojek.com/blog/2020/03/generate-markdown-files/) might be interesting as away to dynamically generate these tables